### PR TITLE
timepicker fixture

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

File: src/app/public/testing/timepicker-fixture.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+   * Set the timepicker's disabled value
+   */
+  public set disabled(value: boolean) {
+    this.getTimepickerInputEl().nativeElement.disabled = value;
+  }
+
+  /**
+   * Flag indicating if timepicker input is valid.
+   */
+  public get valid(): boolean {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/testing/timepicker-fixture.ts:67]: For consistency, I would name this `isValid`.

File: src/app/public/testing/timepicker-fixture.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(this.fixture, this.skyTestId, 'sky-timepicker');
+  }
+
+  /**
+   * The timepicker's currently selected time.
+   */
+  public get time(): string {
```
[the remark this answers is not in the document]
Comment (Blackbaud-ErikaMcVey) [reply src/app/public/testing/timepicker-fixture.ts:34]: This is the getter, I'm assuming you meant to comment on the setter? It set's the input's value, which is the timepicker's value. Should I rename both the getter and setter to `value`?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

File: src/app/public/testing/timepicker-fixture.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Blackbaud-SteveBrush) [reply src/app/public/testing/timepicker-fixture.ts]: I think that would be a good idea, yes.

[the remark this answers is not in the document]
Comment (Blackbaud-ErikaMcVey) [reply src/app/public/testing/timepicker-fixture.ts]: I renamed it in that `address PR feedback` commit

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-timepicker-fixture)

**Pull request merged**